application/hubspot/v0/config/tasks.json

```diff
@@ -0,0 +1,741 @@
+{
+  "$defs":{
+    "ownerId": {
```

Comment:
we should use `kebab-case` for all properties.

Reply:
fixed